content/about/clubs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Affiliated Clubs
```

Comment:
Instead of saying "affiliated", we should probably say "partner" clubs

Reply:
Sounds good, will change!